asyncsleepiq/fuzion/foundation.py

```diff
@@ -17,25 +17,26 @@
     Speed,
 )
 from ..foundation import SleepIQFoundation
-from ..preset import SleepIQPreset
+from .preset import SleepIQFuzionPreset
 from .actuator import SleepIQFuzionActuator
 from .light import SleepIQFuzionLight
 
 FEATURE_NAMES = [
-    "pressureControlEnabledFlag"
-    "articulationEnableFlag"
-    "underbedLightEnableFlag"
-    "rapidSleepSettingEnableFlag"
-    "thermalControlEnabledFlag"
-    "rightHeadActuator"
-    "rightFootActuator"
-    "leftHeadActuator"
-    "leftFootActuator"
-    "flatPreset"
-    "favoritePreset"
-    "snorePreset"
-    "zeroGravityPreset"
-    "watchTvPreset"
+    "TODO", # Not sure what best to call this, but there's one flag at the start of the list that's (from testing) always "dual".

```

Comment:
It's probably the type of bed for "single" vs "dual".  Can name it something more descriptive or even "dual" if it would never change